layer/projects/project_runner.py

```diff
@@ -160,6 +165,33 @@ def run(self, debug: bool = False, printer: Callable[[str], Any] = print) -> Run
                     )
         return run
 
+    async def _upload_tar_packages(self, client: LayerClient) -> None:
+        trace_config = aiohttp.TraceConfig()
```

Comment:
`aiohttp.TraceConfig` is used in resource upload to report the progress, it is not required in this case.